installer/functions/account/get-account-name.ts

```diff
@@ -0,0 +1,33 @@
+import {
```

Comment:
i'm seeing a lot of similarities between this and `get-account-info`... wondering if you could create a common `getActiveAccount` service method to use in both of these functions?